client/components/Histogram.jsx

```diff
@@ -0,0 +1,144 @@
+            <i className="fa fa-star"></i>
+            <div className="ebay-review-item-l">
+              <p className="ebay-review-item-stars">3</p>
+            </div>
+            <div className="ebay-review-item-r">
+              <i className="r-list-bg">
+                <u className="r-list-fc"></u>
+              </i>
+              <span className="review-span">0</span>
+            </div>
+          </div>
+        </li>
+        <li role="img" className="ebay-review-item">
+          <div className="ebay-reviews-item-href">
+            <i className="fa fa-star"></i>
+            <div className="ebay-review-item-l">
+              <p className="ebay-review-item-stars">2</p>
+            </div>
+            <div className="ebay-review-item-r">
+              <i className="r-list-bg">
+                <u className="r-list-fc"></u>
+              </i>
+              <span className="review-span">0</span>
+            </div>
+          </div>
+        </li>
+        <li className="ebay-review-item">
+          <div className="ebay-reviews-item-href">
+            <i className="fa fa-star"></i>
+            <div className="ebay-review-item-l">
+              <p className="ebay-review-item-stars">1</p>
+            </div>
+            <div className="ebay-review-item-r">
+              <i className="r-list-bg">
+                <u className="r-list-fc"></u>
+              </i>
+              <span className="review-span">0</span>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div className="pie-container">
+      <div role="img" className="en pie-wrpr" aria-label="100% Would recommend">
+        <div className="pie">
+          <div className="left">
+            <span className="pie-span"></span>
+          </div>
+          <div className="right">
+            <span className="pie-span"></span>
+          </div>
+        </div>
+        <p className="pie-txt" data-text="agree">
+          <span>Would recommend</span>
+        </p>
+      </div>
+      <div role="img" className="en pie-wrpr" aria-label="100% Good value">
+        <div className="pie">
+          <div className="left">
+            <span className="pie-span"></span>
+          </div>
+          <div className="right">
+            <span className="pie-span"></span>
+          </div>
+        </div>
+        <p className="pie-txt" data-text="agree">
+          <span>Good value</span>
+        </p>
+      </div>
+      <div role="img" className="en pie-wrpr" aria-label="100% Good quality">
+        <div className="pie">
+          <div className="left">
+            <span className="pie-span"></span>
+          </div>
+          <div className="right">
+            <span className="pie-span"></span>
+          </div>
+        </div>
+        <p className="pie-txt" data-text="agree">
+          <span className="pie-span">Good quality</span>
+        </p>
+      </div>
+    </div>
+  </div>
+      )
+
+export default Histogram;
```

Comment:
This is very difficult to read. The format is fine however, I cannot understand what this piece of code is doing. I believe that you could make functions to insert a lot of this repeat code with the slight variable changes you want instead of copying  it verbatim from ebay. Are you planning on changing this to take dynamic variables, I would also break this into multiple components.